src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.tsx

```diff
@@ -49,6 +49,11 @@ describe('InfiniteScroll', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('basic renders outside grommet wrapper', () => {
+    const { container } = render(<InfiniteScroll />);
+    expect(container.firstChild).toMatchSnapshot();
+  });
```

Comment:
Since infinite scroll doesn't have any styling behavior it seems like we don't need this test